openapi/components/schemas/GatewayAccountConfig/Directa24.yaml

```
@@ -59,3 +59,6 @@ allOf:
           useV3Api:
             type: boolean
             description: Specifies whether to use Directa24 API version 3.
+          storeIdForLater:
+            type: boolean
+            description: Specifies whether to store the ID for later use.
```

Alternative suggestion:

```suggestion
          storeIdNumber:
            type: boolean
            description: Specifies whether to store the customer ID number for future use.
```